ToDo/src/css/todo/index.css

```diff
@@ -0,0 +1,26 @@
+/* GENERAL STYLES */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(195, 142, 166);
+}
+
+/* CONTAINER */
+.todo-container {
+  background-color: rgb(250, 250, 250);
+  border-radius: 4px;
+  width: 80vw;
+  min-height: 240px;
+  max-height: 80vh;
+  border: 1px solid white;
```

Comment:
border is unneccessary here it has the same color, remove it. 
p.s. use hex or rgb color

Reply:
"use hex or rgb color" -you mean don't use words like "white/black", only numbers?

Reply:
deleted border